syntax = "proto3";

package message;

service MessageService {
  rpc AddMessage (MessageRequest) returns (MessageResponse);
}

message MessageRequest {
  string message = 1;
}

message MessageResponse {
  string result = 1;
}
